studio/components/grid/query/Query.ts

```diff
@@ -0,0 +1,18 @@
+import { IQueryAction, QueryAction } from './QueryAction';
+
+interface IQuery {
+  from: (table: string, schema?: string) => IQueryAction;
+}
+
+export class Query implements IQuery {
+  /**
+   * @param name    the table name.
+   * @param schema  the table schema, by default set to 'public'.
+   */
+  from(name: string, schema?: string) {
```

Comment:
@phamhieu this is giving me a ts error. Wondering if you've got any ideas.

`Property 'from' in type 'Query' is not assignable to the same property in base type 'IQuery'.`

Reply:
Need to fix QueryAction.insert values param to `Dictionary<any> | Dictionary<any>[]` as the interface